<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1813</width>
    <height>1109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #E0FFFF;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="move_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #7FFFD4;
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;
    border-color: #2F4F4F;
    font: 12px;
    min-width: 10em;
    padding: 3px;
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="movex_label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="movex_slider">
       <property name="minimum">
        <number>-500</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="movex_line">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="movey_label">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="movey_line">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="movez_label">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="movez_line">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="movey_slider">
       <property name="minimum">
        <number>-500</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSlider" name="movez_slider">
       <property name="minimum">
        <number>-500</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="move_label">
       <property name="text">
        <string>Расположение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="rotate_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #7FFFD4;
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;
    border-color: #2F4F4F;
    font: 12px;
    min-width: 10em;
    padding: 3px;
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="rotatex_label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="rotatex_slider">
       <property name="minimum">
        <number>-500</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="rotatex_line">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="rotatey_labe">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="rotatey_line">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="rotatez_label">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="rotatez_line">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="rotatey_slider">
       <property name="minimum">
        <number>-500</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSlider" name="rotatez_slider">
       <property name="minimum">
        <number>-500</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="rotate_label">
       <property name="text">
        <string>Вращение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="zoom_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #7FFFD4;
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;
    border-color: #2F4F4F;
    font: 12px;
    min-width: 10em;
    padding: 3px;
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QSlider" name="zoom_slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="zoom_label">
       <property name="text">
        <string>Масштаб</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="zoom_line">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="vertices_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>531</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #7FFFD4;
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;
    border-color: #2F4F4F;
    font: 12px;
    min-width: 10em;
    padding: 3px;
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="vertices_label">
       <property name="text">
        <string>Вершины</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="vertices_red_label">
       <property name="text">
        <string>Красный</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="vertices_green_label">
       <property name="text">
        <string>Зеленый</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="vertices_blue_label">
       <property name="text">
        <string>Синий</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="vertices_red"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="vertices_green"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="vertices_blue"/>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="vertices_color_label">
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QRadioButton" name="vertices_square">
       <property name="text">
        <string>Квадраты</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QRadioButton" name="vertices_circle">
       <property name="text">
        <string>Круги</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QRadioButton" name="vertices_no">
       <property name="text">
        <string>Нет</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Edges" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>850</y>
      <width>531</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #7FFFD4;
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;
    border-color: #2F4F4F;
    font: 12px;
    min-width: 10em;
    padding: 3px;
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="edges_label">
       <property name="text">
        <string>Линии</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="edges_blue"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="edges_blue_label">
       <property name="text">
        <string>Синий</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="edges_red"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="edges_red_label">
       <property name="text">
        <string>Красный</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edges_green"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="edges_green_label">
       <property name="text">
        <string>Зеленый</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="edges_color_label">
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QRadioButton" name="edges_solid">
       <property name="text">
        <string>Сплошной</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QRadioButton" name="edges_dashed">
       <property name="text">
        <string>Штриховой</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>1251</width>
      <height>1051</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="proection_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #7FFFD4;
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;
    border-color: #2F4F4F;
    font: 12px;
    min-width: 10em;
    padding: 3px;
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="2" column="0" colspan="2">
      <widget class="QRadioButton" name="proection_central">
       <property name="text">
        <string>Центральная</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="proection_label">
       <property name="text">
        <string>Проекция</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QRadioButton" name="proection_parallel">
       <property name="text">
        <string>Параллельная</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1813</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Сохранить как...</string>
     </property>
     <addaction name="actionJPG_2"/>
     <addaction name="actionPNG_2"/>
     <addaction name="actionGIF_2"/>
    </widget>
    <addaction name="action_2"/>
    <addaction name="menu_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionJPG">
   <property name="text">
    <string>JPG</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionGIF">
   <property name="text">
    <string>GIF</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Файл</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionJPG_2">
   <property name="text">
    <string>JPG</string>
   </property>
  </action>
  <action name="actionPNG_2">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionGIF_2">
   <property name="text">
    <string>GIF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
